lib/manage-prototype-handlers.test.js

```diff
@@ -228,16 +247,12 @@ describe('manage-prototype-handlers', () => {
     const pluginDisplayName = { name: 'Test Package' }
     const templatePath = '/template'
     const encodedTemplatePath = encodeURIComponent(templatePath)
-    const view = 'Test View'
     const chosenUrl = '/chosen-url'
-    let mockSend
 
     beforeEach(() => {
-      mockSend = jest.fn()
-      res.status = jest.fn().mockReturnValue({ send: mockSend })
+      res.status = jest.fn().mockReturnValue({ send: res.send })
```

Comment:
Is there any reason we're not using:

```
res.status = jest.fn().mockReturnValue(res)
```

Reply:
No not at all, loads of ways to do this

Let me know if it's a blocker

Reply:
I've added it this morning, nice improvement should we need it in future

There's another handy shorthand if it's helpful for other tests:

```patch
- lookupPackageInfo: jest.fn().mockImplementation((packageName) => {
+ lookupPackageInfo: jest.fn((packageName) => {
```